ansible/group_vars/all

```diff
@@ -16,6 +16,7 @@ runtimesManifest:
     default: true
   python:
   - kind: "python"
```

Comment:
general comment (not specific to this pr):
we might consider adding python:2 as well, make it explicit. 
we should adopt it as a policy, a naked family name is same as :default or disallow altogether.
without versions, we're asking for more headaches in the future.

fyi @bjustin-ibm @jasonpet